frontend/javascripts/oxalis/view/components/markdown_modal.tsx

```diff
@@ -77,7 +78,14 @@ export function MarkdownModal({
               minRows: 5,
               maxRows: 20,
             }}
-            isTextArea
+
+            // style={title == null ? style : undefined}
+            // onChange={this.handleChange}
+            // onFocus={this.handleFocus}
+            // onBlur={this.handleBlur}
+            // value={this.state.currentValue}
+            // onPressEnter={onPressEnter}
+            // onKeyDown={this.blurOnEscape}
```

Comment:
why is this commented?

Reply:
Ups. I need to remove this. Just wanted to compare the props supported by <InputComponent> and <TextArea> and double check that no callback was lost.